web/app/components/single-tab.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  actions: {
+    /** bubble this action up **/
```

Comment:
In general, best practices are to document the code. A good starting point would be with JSDocs

http://usejsdoc.org/about-getting-started.html